vocs.config.ts

```diff
@@ -0,0 +1,71 @@
+import { defineConfig } from "vocs";
+
+export default defineConfig({
+  title: "Open Frames",
+  sidebar: [
+    {
+      text: "Specification",
+      link: "/",
+    },
+    {
+      text: "Quickstart",
+      collapsed: false,
+      items: [
+        {
+          text: "frames.js",
+          link: "/quickstart/frames-js",
+        },
+        {
+          text: "OnchainKit",
+          link: "/quickstart/onchainkit",
+        },
+      ],
+    },
+    {
+      text: "Best Practices",
+      link: "/best-practices",
+    },
+    {
+      text: "Ecosystem",
+      collapsed: false,
+      items: [
+        {
+          text: "XMTP",
+          link: "/ecosystem/xmtp",
+        },
+        {
+          text: "Lens",
+          link: "/ecosystem/lens",
+        },
+        {
+          text: "Farcaster",
+          link: "/ecosystem/farcaster",
+        },
+      ],
+    },
+    {
+      text: "Examples",
+      collapsed: false,
+      items: [
+        {
+          text: "Awesome Open Frames ↗︎",
+          link: "https://github.com/open-frames/awesome-open-frames",
+        },
+      ],
+    },
+  ],
+  socials: [
+    {
+      icon: "github",
+      link: "https://github.com/open-frames/standard",
+    },
+    {
+      icon: "warpcast",
```

Comment:
maybe should be a farcaster logo

Reply:
icon support is quite limited 
<img width="560" alt="Screenshot 2024-04-16 at 23 40 41" src="https://github.com/open-frames/docs/assets/5469870/876baae2-9592-4b64-8fe7-104c793cb73e">

Reply:
<img width="562" alt="Screenshot 2024-04-16 at 23 51 33" src="https://github.com/open-frames/docs/assets/5469870/0a6711fe-6d44-4fec-be58-0a863c4952ca">

i've converted the social icons to links. lmk what you think @davidfurlong